.vsts-ci.yml

```
@@ -10,6 +10,7 @@ variables:
   - name: Codeql.TSAEnabled
     value: true
   - group: DotNet-Sign-SDLValidation-Params
+  - template: /eng/common/templates-official/variables/pool-providers.yml
```

I think you might need to add @self at the end of this since you have an extends statement in the yaml, but I'm not 100% of the rules for that. Running a test build would let you know for sure.
```suggestion
  - template: /eng/common/templates-official/variables/pool-providers.yml@self
```

---

I wasn't sure either.

See [this](https://github.com/dotnet/installer/pull/19016/files#diff-fb7d8d37fd9e28ca688fd102422af3e1d1df13a50d7f2deb2e145aa5ef449057R36) and the "Update pool information" section in the PR description.  The author didn't use `@self`.

---

Seems reasonable!